docs/en/sdk-manual/index.md

```diff
@@ -5,6 +5,8 @@
 - [Swift](./swift.md)  
 - [Web](./web.md)
 
+In addition, we also provide [HTTP API](./http-api.md) for directly report click stream data to the data pipeline.
```

Comment:
--> In addition, we also provide [HTTP API](./http-api.md) to collect clickstream data from other platforms (e.g., server) through http request.